src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php

```diff
@@ -234,6 +239,9 @@ public function retrieve($sessionId)
             $stream->open();
             $access = $this->createBinaryStreamAccess($stream);
 
+            // -- Consume version (not used yet).
```

Comment:
May we expect issues when retrieving sessions that were saved without version?

Reply:
Yes, we will release this as a breaking change.